src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsCaseMappingTest.java

```diff
@@ -0,0 +1,460 @@
+import static uk.gov.hmcts.reform.sscs.helper.mapping.HearingsCaseMapping.CASE_SUB_TYPE;
+import static uk.gov.hmcts.reform.sscs.helper.mapping.HearingsCaseMapping.CASE_TYPE;
+
+class HearingsCaseMappingTest extends HearingsMappingBase {
+
+    @DisplayName("When a valid hearing wrapper is given buildHearingCaseDetails returns the correct Hearing Case Details")
+    @Test
+    void buildHearingCaseDetails() {
+        // TODO Finish Test when method done
+        List<CcdValue<OtherParty>> otherParties = new ArrayList<>();
+        otherParties.add(new CcdValue<>(OtherParty.builder()
+                .hearingOptions(HearingOptions.builder().build())
+                .appointee(Appointee.builder().build())
+                .rep(Representative.builder().build())
+                .build()));
+        otherParties.add(new CcdValue<>(OtherParty.builder()
+                .hearingOptions(HearingOptions.builder().build())
+                .appointee(Appointee.builder().build())
+                .rep(Representative.builder().build())
+                .build()));
+        SscsCaseData caseData = SscsCaseData.builder()
+                .ccdCaseId(String.valueOf(CASE_ID))
+                .benefitCode(BENEFIT_CODE)
+                .issueCode(ISSUE_CODE)
+                .caseCreated(CASE_CREATED)
+                .workAllocationFields(WorkAllocationFields.builder()
+                        .caseNameHmctsInternal(CASE_NAME_INTERNAL)
+                        .caseNamePublic(CASE_NAME_PUBLIC)
+                        .build())
+                .appeal(Appeal.builder()
+                        .hearingOptions(HearingOptions.builder().build())
+                        .appellant(Appellant.builder()
+                                .appointee(Appointee.builder().build())
+                                .build())
+                        .rep(Representative.builder().build())
+                        .build())
+                .otherParties(otherParties)
+                .caseManagementLocation(CaseManagementLocation.builder()
+                        .baseLocation(EPIMS_ID)
+                        .region(REGION)
+                        .build())
+                .build();
+        HearingWrapper wrapper = HearingWrapper.builder()
+                .caseData(caseData)
+                .caseData(caseData)
+                .build();
+
+        CaseDetails caseDetails = HearingsCaseMapping.buildHearingCaseDetails(wrapper);
+
+        assertNotNull(caseDetails.getCaseId());
+        assertNotNull(caseDetails.getCaseDeepLink());
+        assertNotNull(caseDetails.getHmctsInternalCaseName());
+        assertNotNull(caseDetails.getPublicCaseName());
+        assertNotNull(caseDetails.getCaseCategories());
+        assertNotNull(caseDetails.getCaseManagementLocationCode());
+        assertNotNull(caseDetails.getCaseSlaStartDate());
+    }
+
+    @DisplayName("getServiceCode Test")
+    @Test
+    void getServiceCode() {
```

Comment:
Are these TODOs still valid? I don't think you need to test getters

Reply:
I'll remove the TODOs from the ones Andrew has confirmed.

Reply:
Okay, I'll remove the test